rrfs-test/IODA/python/bufr2ioda_nexrad.py

```diff
@@ -0,0 +1,358 @@
+
+    # General informaton
+    converter = 'BUFR to IODA Converter'
+    platform_description = 'NEXRAD'
+
+    logger.info(f"reference_time = {reference_time}")
+
+    bufrfile = f"{cycle_type}.t{hh}z.{data_type}.tm00.{data_format}"
+    DATA_PATH = os.path.join(dump_dir, bufrfile)
+    if not os.path.isfile(DATA_PATH):
+        logger.info(f"DATA_PATH {DATA_PATH} does not exist")
+        return
+    logger.debug(f"The DATA_PATH is: {DATA_PATH}")
+
+    # ============================================
+    # Make the QuerySet for all the data we want
+    # ============================================
+    start_time = time.time()
+
+    logger.info('Making QuerySet')
+    q = bufr.QuerySet(subsets)
+
+    # ObsType
+    #q.add('observationType', '*/TYP')
+
+    # MetaData
+    q.add('year', '*/YEAR')
+    q.add('month', '*/MNTH')
+    q.add('day', '*/DAYS')
+    q.add('hour', '*/HOUR')
+    q.add('minute', '*/MINU')
+    q.add('second', '*/SECO')
+    q.add('latitude', '[*/CLATH, */CLAT]')
+    q.add('longitude', '[*/CLONH, */CLON]')
+    q.add('stationIdentification', '*/SSTN')
+    q.add('height', '*/HSMSL')
+    q.add('heightOfAntenna', '*/HSALG')
+    q.add('volumeIndex', '*/VOID')
+    q.add('scanIndex', '*/SCID')
+    q.add('ppiVolume', '*/VOCP')
+
+    # ObsValue
+    q.add('beamAzimuthAngle', '*/ANAZ')
+    q.add('beamTiltAngle', '*/ANEL')
+    q.add('gateRange', '*/NL2RW{1}/DIST125M')
+    q.add('radialVelocity', '*/NL2RW{1}/DMVR')
+    q.add('spectralWidth', '*/NL2RW{1}/DVSW')
+    q.add('unfoldingVelocity', '*/HNQV')
+
+    # QualityMarker
+    q.add('radialVelocityQM', '*/QCRW')
+
+    end_time = time.time()
+    running_time = end_time - start_time
+    logger.debug(f'Running time for making QuerySet : {running_time} seconds')
+
+    # ==============================================================
+    # Open the BUFR file and execute the QuerySet to get ResultSet
+    # Use the ResultSet returned to get numpy arrays of the data
+    # ==============================================================
+    start_time = time.time()
+
+    logger.info('Executing QuerySet to get ResultSet')
+    with bufr.File(DATA_PATH) as f:
+        try:
+            r = f.execute(q)
+        except Exception as err:
+            logger.info(f'Return with {err}')
+            return
+
+    # ObsType
+    logger.debug(" ... Executing QuerySet for NEXRAD: get ObsType ...")
+    #obstyp = r.get('observationType', type='int32')
+    logger.info('Executing QuerySet: get metadata')
+
+    # MetaData
+    dvsw = r.get('spectralWidth', 'spectralWidth')
+    clath = r.get('latitude', 'spectralWidth')
+    clonh = r.get('longitude', 'spectralWidth')
+    sstn = r.get('stationIdentification', 'spectralWidth')
+    hsmsl = r.get('height', 'spectralWidth')
+    hsalg = r.get('heightOfAntenna', 'spectralWidth')
+    void     = r.get('volumeIndex', 'spectralWidth')
+    scid     = r.get('sacnIndex', 'spectralWidth')
```

Comment:
Is this a typo? `scanIndex`

Reply:
@delippi thanks for catching this typo! Fixed.